manifests/e/ElementTechnologie/openElement/1.57.9/ElementTechnologie.openElement.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ElementTechnologie.openElement
+PackageVersion: 1.57.9
+InstallerType: portable

```

```suggestion
InstallerType: exe
```

This is an installer wizard created by _Advanced Installer_. It's not a portable executable.